### PR TITLE
Add: Assembly Language Course

### PR DESCRIPTION
**Pull request by maruf-pfc**
## What does this PR do?
**Assembly Language Bangla Course**  added to the course file.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -49,6 +51,9 @@
 * [Android Tutorials Material Design](https://www.youtube.com/playlist?list=PLgH5QX0i9K3ru-TfN-YsRWKe4EEOLrWjn) - Anisul Islam
 * [Java, OOP & Android tutorials for beginners in Bengali](https://www.youtube.com/playlist?list=PLV3rqOvr9vgkmELwlSouvJtROQ6MWRbIH) - Zulkarnine Mahmud
 
+### Assembly Language
+
+* [Assembly Language Bangla Tutorial](https://youtube.com/playlist?list=PLEYW3pZS6IQ8UMvusEVnIJ2dDieXKyKRM&si=TWVqCXX-SJ5clOYU) - Maruf Sarker
```
Comment (eshellman) [inline courses/free-courses-bn.md:56]:
```suggestion
* [Assembly Language Bangla Tutorial](https://youtube.com/playlist?list=PLEYW3pZS6IQ8UMvusEVnIJ2dDieXKyKRM) - Maruf Sarker
```

**Comment by maruf-pfc** [comment]
Finally fix the linter error 😀.

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -200,6 +208,11 @@
 * [Angularjs bangla tutorial basic](https://www.youtube.com/playlist?list=PLbC4KRSNcMnr2ZFQne_jotsiX9hGwLJHG) - Learn Hunter
 
 
+#### Electron
+
+* [Electron JS Bangla Tutorials](https://youtube.com/playlist?list=PL_XxuZqN0xVDgr7KreI5PaVZuG8Sx3L2c&si=FOsXyD-fC87a45YJ) - Stack Learner
```
Comment (eshellman) [inline courses/free-courses-bn.md:213]:
```suggestion
* [Electron JS Bangla Tutorials](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDgr7KreI5PaVZuG8Sx3L2c&si=FOsXyD-fC87a45YJ) - Stack Learner
```

File: courses/free-courses-bn.md [hunk 3]
```diff
@@ -74,6 +81,7 @@
 * [C++ Bangla Tutorial for Beginners - 2022](https://www.youtube.com/playlist?list=PLeqnvPK4PpyWsjZvgLTRcc-dkPQXc8SHc) - Shikkhangon BD
 * [C++ STL (Bangla)](https://youtube.com/playlist?list=PLgLCjVh3O6Sgux985GYG22xkFt9z9Sq0_) - LoveExtendsCode
 * [Object Oriented C++ \| Bangla Tutorial](https://www.youtube.com/playlist?list=PLy7uM3PHzMF1hnqhFGE4_A8qTUfFmZ_3y) - Online School
+* [Standard Template Library of C++ (STL) [Bangla]](https://youtube.com/playlist?list=PLoa_roVVsxA0D1Kv_T7rbGHtSdYIUo4f5) - CPS Academy
```
Comment (eshellman) [inline courses/free-courses-bn.md:84]:
```suggestion
* [Standard Template Library of C++ (STL)](https://www.youtube.com/playlist?list=PLoa_roVVsxA0D1Kv_T7rbGHtSdYIUo4f5) - CPS Academy
```

File: courses/free-courses-bn.md [hunk 4]
```diff
@@ -50,6 +52,11 @@
 * [Java, OOP & Android tutorials for beginners in Bengali](https://www.youtube.com/playlist?list=PLV3rqOvr9vgkmELwlSouvJtROQ6MWRbIH) - Zulkarnine Mahmud
 
 
+### Assembly Language
+
+* [Assembly Language Bangla Tutorial](https://youtube.com/playlist?list=PLEYW3pZS6IQ8UMvusEVnIJ2dDieXKyKRM) - Maruf Sarker
```
Comment (eshellman) [inline courses/free-courses-bn.md:57]:
```suggestion
* [Assembly Language Bangla Tutorial](https://www.youtube.com/playlist?list=PLEYW3pZS6IQ8UMvusEVnIJ2dDieXKyKRM) - Maruf Sarker
```

**Review by eshellman — CHANGES_REQUESTED** [review]
It's best to use separate braches for each PR

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**